app/views/users/nbp_wallet/connect.html.slim

```diff
@@ -0,0 +1,24 @@
+.header.mb-5
+  .std-heading
+    i.fa-solid.fa-wallet
+    =< t('.header')
+
+.container
+  .row
+    .col-8.col-md-4.col-lg-3
+      img.nbp_wallet_qr_code[src="#{nbp_wallet_qr_code_users_path}?truncated_reference=#{@truncated_reference}"]
+      br
+      br
+      = button_tag class: 'btn btn-primary regenerate-qr-code-button w-100' do
+        = t('.regenerate_code')
+    .col-8.col-md-6.m-3
+      p.fs-6
+        = t('.info_html', alternative_link: @template_url, app_store_link: 'https://apps.apple.com/de/app/mein-bildungsraum-wallet/id6467007352', play_store_link: 'https://play.google.com/store/apps/details?id=de.bildungsraum.wallet.beta')
+
+br
+br
+hr
```

Comment:
Rather than using `br` tags for styling, let's add here (and above) an ordinary margin:

```suggestion
hr.mt-5
```